signing/secp256k1/impl/src/test/java/tech/pegasys/signers/secp256k1/filebased/FileBasedTransactionSignerTest.java

```diff
@@ -51,11 +51,10 @@ public void success() throws IOException {
     final File keyFile = new File(fileName);
     final File pwdFile = createFile(MY_PASSWORD);
 
-    final TransactionSigner signer =
-        FileBasedSignerFactory.createSigner(keyFile.toPath(), pwdFile.toPath());
+    final Signer signer = FileBasedSignerFactory.createSigner(keyFile.toPath(), pwdFile.toPath());
 
     assertThat(signer).isNotNull();
-    assertThat(signer.getAddress()).isNotEmpty();
+    assertThat(signer.getPublicKey()).isNotNull();
```

Comment:
This seems like a weak test. But maybe it's ok because we test that public key is what we expect elsewhere?

Reply:
I suspect the problem is that the WalletUtils doesn't tell you what the publickey is - this just proves that one was created :/ Yeah, a bit weak... but maybe not worth chasing?

Reply:
no don't think it is